src/h3/api/_api_template.py

```diff
@@ -223,28 +336,53 @@ def polyfill(geojson, res, geo_json_conformant=False):
         return _out_unordered(mv)
 
     def h3_is_pentagon(h):
-        """
-        a pentagon should still pass is_cell(), right?
-        :returns: boolean
+        """ Returns `True` if input is a valid H3 Cell which is
+        a pentagon.
+
+        todo: a pentagon should still pass is_cell(), right?
+
+        Returns
+        -------
+        bool
         """
         return _cy.is_pentagon(_in_scalar(h))
 
     def h3_get_base_cell(h):
-        """
-        :returns: boolean
+        """ Return the parenting cell of resolution `0`.
+
+        todo/question: overlap with the `get_parent()` function?
+
+        Returns
+        -------
+        bool
```

Comment:
This returns an integer base cell number, not a bool (and also not a parent index - it just reads a bitfield from the index.)

Reply:
@isaacbrodsky, by "base cell number", do you mean the number of the face of the icosahedron?

Reply:
The terminology is a little confusing, because here (https://github.com/uber/h3/blob/master/src/h3lib/include/h3api.h.in#L315-L321), "base cell" seems to mean the face integer number.

But here, (https://github.com/uber/h3/blob/master/src/h3lib/include/h3api.h.in#L285-L294), the comment seems to suggest that "base cell" is equivalent to "resolution 0 index".

Reply:
OK, I read the docs a bit closer. I see the difference now. This returns a base cell number from 0 to 121, and the base cell H3 indices would have that number encoded in the H3Index for a cell with resolution zero.

What would be good terminology to distinguish between the 0--121 "base cell" and the H3Index "base cell" representations?

Reply:
Updated the description. Let me know what you think.